image/discourse_bench/Dockerfile

```diff
@@ -17,11 +19,12 @@ RUN gem install facter &&\
 
 # reconfigure PostgreSQL template databases to use UTF-8
 # otherwise rake db:create fails
-RUN pg_dropcluster --stop 9.3 main &&\
-    pg_createcluster --start -e UTF-8 9.3 main
+RUN pg_dropcluster --stop 9.5 main &&\
```

Comment:
https://github.com/discourse/discourse/pull/4454 allows running the benchmarks via discourse/discourse_bench docker container.